ddtrace/context.py

```diff
@@ -26,7 +32,8 @@
         _MetaDictType,  # _meta
         _MetricDictType,  # _metrics
     ]
-
+# regex matches ",", ";", ":", and characters outside the ASCII range 0x20 to 0x7E
```

Comment:
```suggestion
```

this comment just echos the actual definition which means that we have to maintain both and likely won't.